pyrogram/types/messages_and_media/message.py

```diff
@@ -498,6 +501,9 @@ async def _parse(
             elif isinstance(action, raw.types.MessageActionInviteToGroupCall):
                 voice_chat_members_invited = types.VoiceChatMembersInvited._parse(client, action, users)
                 service_type = "voice_chat_members_invited"
+            elif isinstance(action, raw.types.MessageActionSetMessagesTTL):
```

Comment:
Reading this made me think that maybe we should rename the whole thing to `set_chat_messages_ttl` or just `set_chat_ttl` since the feature applies to new messages not to the whole history (or can be seen as a global chat setting regardless of messages or history). What do you think? There's also a `ttl_period` field for messages (see the raw Message object) which I guess it will be set for all new messages when ttl is enabled.

Reply:
So, `set_chat_ttl()`, `Chat.set_ttl()`, `Message.chat_ttl_period`.